wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
-[Multi_mods_host image]: ./Multi_mods_host.jpg "Host (osu!) mods options"
-[Multi_mods_player image]: ./Multi_mods_player.jpg "Player (osu!) mods options (With &quot;Free Mods&quot; on)"
-[Multi_host_song_change image]: ./Multi_host_song_change.jpg "When the Host is changing songs, this bar will become a temporary placeholder."
-[Multi_SS image]: ./Multi_SS.jpg "Example of how the Song Selection will look like for host."
-[Multi_mh_single image]: ./Multi_mh_single.jpg "Example of Match History."
-[Multi_mh_team image]: ./Multi_mh_team.jpg "Example of Team Match History."
-[osu! icon link]: /wiki/shared/Osu.gif
-[osu!taiko icon link]: /wiki/shared/Taiko.gif
-[osu!catch icon link]: /wiki/shared/Ctb.gif
-[osu!mania icon link]: /wiki/shared/Mania.gif
-[Multi_PB_B image]: ./Multi_PB_B.jpg "Blue"
-[Multi_PB_R image]: ./Multi_PB_R.jpg "Purple-Red"
-[Multi_PB_F image]: ./Multi_PB_F.jpg "Failed/Grey"
-[Multi_PB_TAG image]: ./Multi_PB_TAG.jpg "Tag/Green"
-[Multi_PB_SKIP image]: ./Multi_PB_SKIP.jpg "Skipped"
-[Multi_PB_QUIT image]: ./Multi_PB_QUIT.jpg "Quit"
+[Beatmaps_peppy image]: /wiki/shared/Beatmaps_peppy.jpg "Official image on beatmap extraction."
+[Multi_lobby image]: ./img/Multi_lobby.jpg "Example of a typical day in the Multiplayer Lobby"
+[Multi_new_game image]: ./img/Multi_new_game.jpg "Setting up configuration for a New Game"
+[Multi_roomhost image]: ./img/Multi_roomhost.jpg "Example of Host Match Setup"
+[Multi_roomplayer image]: ./img/Multi_roomplayer.jpg "Example of Player Match Setup (With Free Mods on)"
+[Multi_mods_host image]: ./img/Multi_mods_host.jpg "Host (osu!) mods options"
+[Multi_mods_player image]: ./img/Multi_mods_player.jpg "Player (osu!) mods options (With Free Mods on)"
+[Multi_beatmapinfo image]: ./img/Multi_beatmapinfo.jpg "Beatmap information when hovered over."
+[Multi_host_song_change image]: ./img/Multi_host_song_change.jpg "When the Host is changing songs, this bar will become a temporary placeholder."
+[Multi_SS image]: ./img/Multi_SS.jpg "Example of how the Song Selection will look like for host."
+[Multi_mh_single image]: ./img/Multi_mh_single.jpg "Example of Match History."
+[Multi_mh_team image]: ./img/Multi_mh_team.jpg "Example of Team Match History."
+
+<!-- Team Mode images -->
+[HTH Interface image]: ./img/Multi_HTH.jpg "Head-to-Head Interface"
+[Team Vs Interface image]: ./img/Multi_team_vs.jpg "Team Vs Interface"
+[Tag Coop Interface image]: ./img/Multi_tag_co-op.jpg "Tag Coop Interface"
+[Tag Team Vs Interface image]: ./img/Multi_tag_team_vs.jpg "Tag Team Vs Interface"
+[HTH Grade image]: ./img/Multi_grade.jpg "Head-to-Head grade screen"
+[Team Vs Grade image]: ./img/Multi_team_grade.jpg "Team Vs grade screen"
+[Tag Coop Grade image]: ./img/Multi_co-op_grade.jpg "Team Co-Op grade screen"
+[Tag Team Vs Grade image]: ./img/Multi_tag_team_grade.jpg "Tag Team Vs grade screen"
 
 <!--Title-->
 # Multiplayer
-![Flowchart of a multi game][Multi_flowchart image]
 
 ## Introduction
```

Comment:
Remove this header, the paragraph introducing a page is already implied to be the "introduction" and/or "lead section"